sesman/chansrv/smartcard_pcsc.c

```diff
@@ -1821,37 +2169,25 @@ my_pcsc_trans_conn_in(struct trans *trans, struct trans *new_trans)
 int
 scard_pcsc_init(void)
 {
-    char *home;
+    const char *csock_name;
     int disp;
     int error;
 
     LLOGLN(0, ("scard_pcsc_init:"));
     if (g_lis == 0)
     {
         g_lis = trans_create(2, 8192, 8192);
-        home = g_getenv("HOME");
-        disp = g_display_num;
-        g_snprintf(g_pcsclite_ipc_dir, 255, "%s/.pcsc%d", home, disp);
-
-        if (g_directory_exist(g_pcsclite_ipc_dir))
+        csock_name = g_getenv("XRDP_PCSCLITE_CSOCK_NAME");
```

Comment:
I can't see this environment variable is defined anywhere. Defined constant in `env.c` is `PCSCLITE_CSOCK_NAME`. No `XRDP_` prefix.